modin/pandas/utils.py

```diff
@@ -104,6 +104,29 @@ def remote(self, *args):
         return result
 
 
+def post_task_gc(func):
+    """Perform garbage collection after the task is executed.
+
+    Usage:
+        ```
+        @ray.remote
+        @post_task_gc
+        def memory_hungry_op():
+            ...
+        ```
+    Note:
+        - This will invoke the GC for the entire process. Expect
+          About 100ms latency. 
+    """
+    def wrapped(*args):
+        result = func(*args)
+
+        import gc
```

Comment:
Why import here and not at the top of the file?

Reply:
This will cause `import gc` to be run every time that a function with the decorator is run

Reply:
Because the ray worker process might not correctly import gc. (Especially working with cloudpickle)

Reply:
@kunalgosar The python process will only import it once, the rest import is noop.

Reply:
I'm not sure if that's true, I think importing it at the beginning of the file would be fine. This seems to work well for the rest of the code?